great_expectations/profiler/parameter_builder/parameter.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import asdict, dataclass
+from typing import Any, Dict, Optional, Union
+
+from great_expectations.core import IDDict
+from great_expectations.core.util import convert_to_json_serializable
+from great_expectations.types import SerializableDictDot
+
+
+@dataclass
```

Comment:
Lovely!

Reply:
👍 Just added a comment to make the refactor to use `DataClass` in develop -- hope it works!